examples/js/loaders/LDrawLoader.js

```diff
@@ -869,7 +860,7 @@ THREE.LDrawLoader = ( function () {
 				case LDrawLoader.FINISH_TYPE_RUBBER:
 
 					// Rubber is best simulated with Lambert
```

Comment:
Could you remove this comment, since MeshLambertMaterial is no longer used?

Reply:
Oh yeah good call, thanks!